docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -202,17 +201,17 @@ All modules and plugins MUST:
 
 * Include a :ref:`DOCUMENTATION <documentation-block>` block.
 * Include an :ref:`EXAMPLES <examples-block>` block (except where not relevant for the plugin type).
-* Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection (including ``ansible.builtin`` for the listed entities from ansible-core.
+* Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection including ``ansible.builtin.`` for ansible-core.
   
 When using ``version_added`` in the documentation:
 
-* Declare the version of the collection in which the options were added -- NOT the version of Ansible.
+* Declare the version of the collection in which the options were added -- NOT the version of Ansible/ansible-core.
 * If you for some reason really have to specify version numbers of Ansible or of another collection, you also have to provide ``version_added_collection: collection_name``. We strongly recommend to NOT do this.
-* Include ``version_added`` when you add new content (modules, plugins, options) to an existing collection. The values are shown in the documentation and can be useful, but you do not need to add ``version_added`` to every option, module, and plugin when creating a new collection.
+* Include ``version_added`` when you add new content (modules, plugins, options, return values) to an existing collection. The values are shown in the documentation and can be useful, but you do not need to add ``version_added`` to every option, module, and plugin when creating a new collection.
```

Comment:
Not sure how to word this but it also applies to config manager sub-sections of plugin options, like `ini`, `vars`, etc.

Reply:
good point, me too, any suggestions are welcome